vere/pier.c

```diff
@@ -782,6 +827,39 @@ _pier_disk_load_commit(u3_pier* pir_u,
       c3_free(buf_d);
 
       ovo = u3ke_cue(u3k(mat));
+
+      //  single-home
+      //
+      if ( (0ULL == pos_d) &&
+           (1ULL == lav_d) )
```

Comment:
Can you explain this condition?

Reply:
`lav_d` is the next event we expect (ie, we have no checkpoint, but we have a log -- we're doing a full log replay), `pos_d` is a cursor into the log itself. The `%boot` command is the first thing in the log.